### DISCUSSION TITLE
Missing the auto_brightness

### DISCUSSION THREAD

**Discussion by krambriw** [Motion-Project/motionplus#125] [Q&A] [answered]
Hey, running Motion for years now, works great , thanks! Was looking to migrate to plus version but I can't find the auto_brightness setting in plus. Is there another way? My cameras need it
Best regards, Walter

**Comment by Mr-Dave** [comment, marked as answer]
The auto_brightness for v4l2 devices was removed from Motionplus.  I'm very surprised that within Motion it performed well enough for you to actually use it.  In all tests I observed, it just oscillated and was terrible.  To adjust the brightness/exposure within Motionplus, you'd need to use the [v4l2_params](https://motion-project.github.io/motionplus_config.html#v4l2_params) parameter and specify the values that are applicable to your camera.

**Comment by krambriw** [comment]
Hi Mr-Dave, yes I can agree on that. I see this flickering as well, especially during transition periods from night to day and vice versa
I guess my cameras, now several years old, should be replaced (some day) to more modern devices that handles the auto brightness themselves
